### ISSUE TITLE
Is the breakout confirmed working?

### ISSUE THREAD

**Issue by MangoIV** [atoomnetmarc/Minew-MS88SF2-breakout#1]
Hi! I would like to ask, whether the breakout works, including USB. Thanks in advance.

**Comment by atoomnetmarc** [comment]
I do not know if everything works. I have not tested it yet.

What I have tested on the V1.0 PCB is JTAG, that works. Some LEDs I have seen flashing. The reset button works (if you enable it). But the user-button and all other pins I have not tested.\
USB did not work on V1.0, but after mounting a bodge wire (VDDH was connected to VDD, but should be connected to 5V), USB started working and after programming a UF2 bootloader I was able drag firmware to the module. In this case it was the FTD variant of the OpenThread project. I was able to connect to my border router.\
The bodge wire resulted in a V1.1 PCB, but I have not made that PCB to validate.

TL;DR: I am not aware of any bugs in V1.1, but I have not tested everything.

**Comment by MangoIV** [comment]
So basically if I have the VDDH connected to 5V over the R2 resistor as in configuration 1 in the datasheet, USB works and the module itself generally comes alive?

**Comment by MangoIV** [comment]
Thanks a lot for your long answer :)

**Comment by atoomnetmarc** [comment]
I have it working with this schematic:

![afbeelding](https://user-images.githubusercontent.com/48124268/110516919-7820be80-810a-11eb-8bd1-957213da28e6.png)

So yes. VDDH to USB 5V (and VDD to 3V3) and it works.

**Comment by atoomnetmarc** [comment]
Added test checklist to README.md https://github.com/atoomnetmarc/Minew-MS88SF2-breakout/commit/54cf1dcff473c9ce5b2402c0c0bcd90490e8f438

**Comment by MangoIV** [comment]
Thanks a lot! I was mainly concerned whether the module was going to come on on USB because the possible configurations aren't well described in the data sheet. Now I guess I'll have to try myself whether it also comes on on battery. Thanks a lot for your patience ;)
